src/default-markup.ts

```diff
@@ -0,0 +1,235 @@
+/**
+ * Only show comment during develop
+ */
+type Comment = (text: string) => string
+const comment: Comment = text => process.env.NODE_ENV !== 'production'
+  ? `<!--${text}-->`
+  : ``
```

Comment:
I really like this! Since it doesn't rely on any props of `defaultMarkup`, I thought it's a good idea to take this out into its own function &make it so that it only add comments in development mode.